bootstrap/sql/mysql/create_tables.sql

```diff
@@ -38,11 +38,14 @@ CREATE TABLE IF NOT EXISTS schema_version_info (
   version          INT                   NOT NULL,
   schemaMetadataId BIGINT                NOT NULL,
   timestamp        BIGINT                NOT NULL,
+  state            TINYINT               NOT NULL,
```

Comment:
Id be really pleased to see https://github.com/hortonworks/registry/pull/232 getting put in before these changes, and then let it manage this upgrade